concepts/problem-solving/README.md

```diff
@@ -1,96 +1,149 @@
 # Problem Solving
 
+> Never trust a computer you can't throw out a window. - Steve Wozniak
+
+## Capabilities
+You are comfortable:
+
+ - Persevering with a tough problem  
+ - Using strategies to solve a tough problem  
+ - Recognising when it's time to ask for help  
+
 ## Overview
-You'll be constantly facing challenging problems! Resolving problems is an art that you'll be practicing at EDA. Tough problems are the best learning opportunities. 
+At EDA (and as a professional developer) you'll be constantly facing challenging problems! Tough problems are the best learning opportunities. 
 
-There are many strategies for resolving problems. The point is that you have a methodical strategy that keeps you calm and productive, so you can avoid a black hole of frustration.
+There are many strategies for resolving problems. Having a methodical strategy keeps you calm and productive, so you can avoid a black hole of frustration.
 
 ## Steps
+
+If your code is not properly formatted before starting this process, _do that first_! We can't emphasise this enough. Properly formatted code will allow you to see your work (and that of others) more clearly. It will make some mistakes obvious, and reduce the chance of making others. Here are some other steps to try:
+
 1. Pseudocode
 2. Try something!
-3. Debug
-  - Rubber ducky
-  - Read error message
-  - Console.log()
-  - Google
-4. Blocked?
-    1. Ask peers on cohort slack
-    2. Tag a coach
-4. Reflect
-
-**1. PseudoCode**    
+3. Simplify
+4. Read error message
+5. Improve visibility of problems using console output
+6. Debuggers: 
+  * Chrome, Firefox or Edge dev tools
```

Comment:
For completeness we should add Safari to this list. Its dev tools aren't bad and quite important when doing iOS Cordova apps.